### PR TITLE
MinGW support

### PR DESCRIPTION
**Pull request by sandman42292**
This PR modifies some codes to enable openpnp-capture to be built with mingw.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -84,6 +86,7 @@ IF (WIN32)
     # add files for WIN32
     target_sources(openpnp-capture PRIVATE win/platformcontext.cpp
                                            win/platformstream.cpp)
+    target_link_libraries(openpnp-capture strmiids)
```
Comment (sandman42292) [inline CMakeLists.txt:89]: I have no idea why current master can be built without strmiids library.

File: win/platformdeviceinfo.h [hunk 2]
```diff
@@ -27,7 +27,7 @@
 #define win_platformdeviceinfo_h
 
 #include <windows.h>
-#include <Dshow.h>
```
Comment (sandman42292) [inline win/platformdeviceinfo.h:30]: Windows doesn't distinguish "Dshow.h" and "dshow.h" but Linux does.

File: win/platformstream.h [hunk 3]
```diff
@@ -31,8 +31,7 @@
 
 #include <windows.h>
 #include <dshow.h>
-#include <Vidcap.h>
```
Comment (sandman42292) [inline win/platformstream.h:34]: "Vidcap.h" seems not needed anymore now.

**Comment by vonnieda** [comment]
Looks good, thank you @sandman42292! Sorry for the long delay on getting this merged!

**Pull request merged**